src/common/telemetry_ops/cluster_telemetry.rs

```diff
@@ -46,28 +47,35 @@ impl From<&Settings> for ClusterConfigTelemetry {
     }
 }
 
-#[derive(Serialize, Clone, Debug, JsonSchema)]
+#[derive(Serialize, Clone, Debug, JsonSchema, Anonymize)]
 pub struct ClusterStatusTelemetry {
+    #[anonymize(false)]
     pub number_of_peers: usize,
+    #[anonymize(false)]
     pub term: u64,
+    #[anonymize(false)]
     pub commit: u64,
+    #[anonymize(false)]
     pub pending_operations: usize,
     pub role: Option<StateRole>,
     pub is_voter: bool,
+    #[anonymize(value = None)]
     pub peer_id: Option<PeerId>,
     pub consensus_thread_status: ConsensusThreadStatus,
 }
 
-#[derive(Serialize, Clone, Debug, JsonSchema)]
+#[derive(Serialize, Clone, Debug, JsonSchema, Anonymize)]
 pub struct ClusterTelemetry {
     pub enabled: bool,
     #[serde(skip_serializing_if = "Option::is_none")]
     pub status: Option<ClusterStatusTelemetry>,
     #[serde(skip_serializing_if = "Option::is_none")]
     pub config: Option<ClusterConfigTelemetry>,
     #[serde(skip_serializing_if = "Option::is_none")]
+    #[anonymize(false)]
     pub peers: Option<HashMap<PeerId, PeerInfo>>,
     #[serde(skip_serializing_if = "Option::is_none")]
+    #[anonymize(false)]
     pub metadata: Option<HashMap<String, serde_json::Value>>,
```

Comment:
I think we should anonymize this as well.

Reply:
Good catch. Using `#[anonymize(value = None)]` is probably good enough to not include it at all.

Reply:
I would still prefer to see if there is <something> in metadata, might be indication of a problem